sandbox.html

```diff
@@ -68,18 +68,25 @@
           hideWorkbookToolBar: (hideWorkbookToolBar === 'on')
         });
       },
-      showMap ({ embeddingTargetId, geometry, mapExtent, locationFinder }) {
+      showMap ({ embeddingTargetId, geometry, mapExtent, locationFinder, useMapSrs }) {
         cadenzaClient.showMap(embeddingTargetId, {
           geometry: geometry && JSON.parse(geometry),
           mapExtent: mapExtent && mapExtent.split(','),
-          locationFinder
+          locationFinder,
+          useMapSrs: useMapSrs === 'on'
         });
       },
-      createGeometry ({ embeddingTargetId, geometryType }) {
-        cadenzaClient.createGeometry(embeddingTargetId, geometryType);
+      createGeometry ({ embeddingTargetId, geometryType, useMapSrs }) {
+        cadenzaClient.createGeometry(embeddingTargetId, geometryType, { useMapSrs: useMapSrs === 'on' });
       },
-      editGeometry ({ embeddingTargetId, geometry }) {
-        cadenzaClient.editGeometry(embeddingTargetId, JSON.parse(geometry));
+      editGeometry ({ embeddingTargetId, geometry, useMapSrs }) {
+        cadenzaClient.editGeometry(embeddingTargetId, JSON.parse(geometry), { useMapSrs: useMapSrs === 'on' });
+      },
+      fetchData ({ embeddingTargetId, mediaType }) {
+        cadenzaClient.fetchData(embeddingTargetId, { accept: mediaType });
```

Comment:
That's what I used it for, but it might be confusing for other devs since nothing seems to happen when when submitting. I thought it's best to leave it out.